### PR TITLE
Callout limitation with convert account

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

Adds a tip to the convert your account docs that the feature is supported for Personal subscriptions.

[Deploy preview](https://deploy-preview-19212--docsdocker.netlify.app/admin/convert-account/)

### Related issues (optional)

Closes JIRA https://docker.atlassian.net/browse/ENGDOCS-1963

### CODE REVIEW

**Review by weswhite — APPROVED** [review]

File: content/admin/convert-account.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ aliases:
...
 
+> **Tip**
+>
+> You can convert an account with a Docker Personal subscription to an organization. If you need to convert an account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
+{ .tip }
+
 You can convert an existing user account to an organization. This is useful if you need multiple users to access your account and the repositories that it’s connected to. Converting it to an organization gives you better control over permissions for these users through [teams](manage-a-team.md) and [roles](roles-and-permissions.md).
 
-When you convert a user account to an organization, the account is migrated to a Team plan that requires a paid subscription. For more information, see [Docker Pricing](https://www.docker.com/pricing).
+When you convert a user account to an organization, the account is migrated to a Team plan.
```
Comment (ob1dev) [inline content/admin/convert-account.md:16]:
```suggestion
When you convert a user account to an organization, the account is migrated to a Docker Team plan.
```

File: content/admin/convert-account.md [hunk 2]
```diff
@@ -6,9 +6,14 @@ aliases:
 - /docker-hub/convert-account/
 ---
 
+> **Tip**
+>
+> You can convert an account with a Docker Personal subscription to an organization. If you need to convert an account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
```
Comment (ob1dev) [inline content/admin/convert-account.md:11]: **suggestion:** I believe in other places, we use the term `your personal account` when referring to a Docker user account.

```suggestion
> You can convert your personal account with a Docker Personal subscription to an organization. If you need to convert your personal account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
```

Comment (ob1dev) [inline content/admin/convert-account.md:11]: **nitpick:** maybe adding a link (https://account.docker.com/) where to convert an account 🤷🏻‍♂️

**Review by ob1dev — APPROVED** [review]
LGTM

File: content/admin/convert-account.md (hunk 2 above)
Comment (stephaurelio) [reply content/admin/convert-account.md:11]: There's instructions later on in the doc for where to go to convert (this is still in Hub for now).

**Review by dvdksn — APPROVED** [review]

**Pull request merged**